tests/Doctrine/Tests/Common/Annotations/AbstractReaderTest.php

```diff
@@ -470,7 +471,7 @@ public function testWillFailOnAnnotationConstantReferenceContainingDashes()
     }
 
     /**
-     * @return AnnotationReader
+     * @return Reader
```

Comment:
I noticed this was wrong thanks to phpstan

Reply:
Perhaps this could be in a different commit 😁

Reply:
You're absolutely right 👍